source/retryable-reads/tests/listIndexNames.yml

```diff
@@ -0,0 +1,60 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data: []
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "ListIndexNames succeeds on first attempt"
+        clientOptions:
+            retryReads: true
+        operation: &retryable_operation
+            name: listIndexNames
+            object: collection
+        expectations:
+            -  &retryable_command_started_event
+                command_started_event:
+                    command:
+                        listIndexNames: *collection_name
+                    database_name: *database_name
+    -
+        description: "ListIndexNames succeeds on second attempt"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands:
+                        - listIndexes
+                closeConnection: true
+        operation: *retryable_operation                
+        expectations:
+             - *retryable_command_started_event
```

Comment:
This test is missing a `*retryable_command_started_event`.

Reply:
Noted, will fix!

Reply:
Still missing:
```suggestion
             - *retryable_command_started_event
             - *retryable_command_started_event
```